core/templates/dev/head/pages/library/Library.js

```diff
@@ -80,6 +80,38 @@ oppia.controller('Library', [
       $http.get('/libraryindexhandler').success(function(data) {
         $scope.libraryGroups = data.activity_summary_dicts_by_category;
 
+        $scope.userIsLoggedIn = GLOBALS.userIsLoggedIn;
+        $scope.activitiesOwned = {explorations: {}, collections: {}};
+        if ($scope.userIsLoggedIn) {
+          $http.get('/creatordashboardhandler/data')
+            .then(function(response) {
+              $scope.libraryGroups.forEach(function(activityGroups) {
+                var activitySummaryDicts =
+                (activityGroups.activity_summary_dicts);
+
+                activitySummaryDicts.forEach(function(activity) {
```

Comment:
Actually the dummy var should be activitySummaryDict, right?

Reply:
Yeah, right!